external/get_collections.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+# =============================================================================
+# Copyright (c) 2020 NVIDIA. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+
+
+import os
+import json
+import argparse
+
+import nemo
+from nemo.utils import logging
+
+# For now - hardcoded list.
+import nemo.collections.common as col_common
+import nemo.collections.asr as col_asr
+import nemo.collections.cv as col_cv
+import nemo.collections.nlp as col_nlp
+import nemo.collections.tts as col_tts
+
+
+
+def analyse_collection(id, col):
+    return {
+        "id": id,
+        "name": col.__name__,
+        "description": col.__description__,
+        "version": col.__version__,
+        "author": col.__author__
```

Comment:
Why do we need author? Almost entire team works on multiple parts of collections so author is not particularly appropriate here.

Reply:
I agree, that is why I added __authors__ = "NVIDIA Corporation" to every collection's `__init__.py`.

Probably will remove this, for now I am just "fulfilling" the requirements that Chris send:

"Script 1: If NeMo has collection metadata (human-readable descriptions, URLs, version strings…) we should retrieve that as well.
Proposed format is an array of dictionaries: [ { “id” : “Collection1” }, { “id” : “Collection2” }, … ]
If more information is available, add additional fields to the dictionary: { “id” : “Collection1”, **“author” : “NVIDIA Corporation”**, … }
We can also move metadata to script 2 if that’s more convenient."